### DISCUSSION TITLE
Remember done exercises

### DISCUSSION THREAD

**Discussion by MauriceHeinze** [JCluzet/42_EXAM#9] [Ideas]
Would be great if the program remembers what exercises I did the last time to get more variation, because right now I get almost every time I use it the same exercises.

**Comment by JCluzet** [comment]
Hey MauriceHeinze 👋

Thank you for your proposal concerning 42_EXAM.

Indeed this option seems interesting, however it would ask the user at each new exam if he wants to redo the previous ones or if he ONLY wants new ones, and that would add new prompt...

If you can think of a way I could add this without being annoying to the users, I would add it.

However, I have added a new command in the last push that might be useful to solve your problem! 
It's the `new_ex` command which allows you to generate a new exercise immediately (in case you've already done it)

Good luck for your exam!

**Reply by MauriceHeinze** [reply]
Great! That's all I needed. 

Regarding your concerns about adding a new prompt, what about having an option in the menu where you can change the settings (like also background color for example)?

Thanks a lot, hope I pass this time :)

**Reply by JCluzet** [reply]
Hi MauriceHeinze 👋

Thank you for your valuable  idea.
I have just released the **v2.1 release** with  the beginning of a menu, that should be what you were talking about.

Have a good exam!
